client/src/App.css

```diff
@@ -205,6 +208,14 @@
     align-items: center; 
 }
 
+.smoothScroll {
```

Comment:
what does smoothScroll do? Is this the parallax-y sorta thing on the home page?

Reply:
Smooth scroll is used to make the scrolling smoother on mobile devices.